<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StabilizationWidget</class>
 <widget class="QWidget" name="StabilizationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stabilization</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TimeSequenceWidget" name="timeSequence" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="stabilizationGroupBox">
     <property name="title">
      <string>1 - Stabilization</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="computeOnRangeButton">
        <property name="text">
         <string>Stabilize sequence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset sequence</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>2 - Orientation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="orientationWarningLabel">
        <property name="text">
         <string notr="true">Orientation cannot be adjusted when using X type of projection</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelWarningIcon">
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../libvideostitch-gui/libvideostitch-gui.qrc">:/assets/images/warning.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="editOrientationButton">
        <property name="text">
         <string>Adjust on current frame</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="vs-checkable-button" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeSequenceWidget</class>
   <extends>QWidget</extends>
   <header>../src/widgets/timesequence.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../libvideostitch-gui/libvideostitch-gui.qrc"/>
 </resources>
 <connections/>
</ui>
